mysql-test/t/ssl_bug75311-master.opt

```diff
@@ -1 +1,6 @@
---ssl-cipher=DHE-RSA-AES256-SHA
+# Let's make this test generic for servers linked with OpenSSL ver < 1.1.1 (without TLSv1.3) and >= 1.1.1 (with TLSv1.3)
+# --tls-ciphersuites= 		- no default TLSv1.3 ciphers
```

Comment:
Formatting (use spaces instead of tabs everywhere)

Reply:
Done